app-backend/datamodel/src/main/scala/Scope.scala

```diff
@@ -258,150 +299,182 @@ object Scopes {
+
+  case object DatasourcesCRUD
+      extends SimpleScope(makeCRUDScopedActions(Domain.Datasources))
+
+  case object ExportsCRUD
+      extends SimpleScope(makeCRUDScopedActions(Domain.Exports))
+
+  case object FeatureFlagsScope
       extends SimpleScope(
-        Set(Action.Read, Action.Update, Action.Delete) map {
-          makeScopedAction(Domain.Uploads, _)
-        }
+        Set(ScopedAction(Domain.FeatureFlags, Action.Read, None))
       )
 
-  case object UploadsCRUD
-      extends ComplexScope(
+  case object LicensesScope
+      extends SimpleScope(Set(ScopedAction(Domain.Licenses, Action.Read, None)))
+
+  case object MapTokensCRUD
+      extends SimpleScope(makeCRUDScopedActions(Domain.MapTokens))
+
+  case object OrganizationAdmin
+      extends SimpleScope(
         Set(
-          Uploader,
-          new SimpleScope(Set(makeScopedAction(Domain.Uploads, Action.Delete)))
+          ScopedAction(Domain.Organizations, Action.AddUser, None),
+          ScopedAction(Domain.Organizations, Action.Update, None)
         )
       )
 
-  case object ScenesCRUD
-      extends SimpleScope(makeCRUDScopedActions(Domain.Scenes))
-
-  case object ScenesMultiPlayer
-      extends SimpleScope(Set(makeScopedAction(Domain.Scenes, Action.Share)))
-
-  case object ScenesFullAccess
-      extends ComplexScope(Set(ScenesCRUD, ScenesMultiPlayer))
+  case object OrganizationsMember
+      extends SimpleScope(
+        Set(Action.ListUsers, Action.Read, Action.Search)
+          .map(makeScopedAction(Domain.Organizations, _))
+      )
 
-  case object ProjectsCRUD
-      extends SimpleScope(makeCRUDScopedActions(Domain.Projects))
+  case object PlatformDomainAdminScope
+      extends SimpleScope(
+        Set(
+          ScopedAction(Domain.Organizations, Action.Create, None),
+          ScopedAction(Domain.Organizations, Action.Delete, None),
+          ScopedAction(Domain.Platforms, Action.AddUser, None),
+          ScopedAction(Domain.Platforms, Action.Create, None),
+          ScopedAction(Domain.Platforms, Action.Delete, None),
+          ScopedAction(Domain.Platforms, Action.ListUsers, None),
+          ScopedAction(Domain.Platforms, Action.Update, None)
+        )
+      )
 
-  case object ProjectExport
-      extends SimpleScope(Set(Action.ListExports, Action.CreateExport) map {
-        makeScopedAction(Domain.Projects, _)
-      })
+  case object PlatformDomainMemberScope
+      extends SimpleScope(
+        Set(ScopedAction(Domain.Platforms, Action.Read, None))
+      )
 
-  case object ProjectAnnotate
+  case object ProjectAnnotateScope
       extends SimpleScope(
         Set(
-          Action.CreateAnnotation,
-          Action.DeleteAnnotation,
-          Action.UpdateAnnotation,
-        ) map {
-          makeScopedAction(Domain.Projects, _)
-        }
+          ScopedAction(Domain.Projects, Action.CreateAnnotation, None),
+          ScopedAction(Domain.Projects, Action.CreateTaskGrid, None),
```

Comment:
This is for non-Groundwork users right? I'm curious what an RF platform user needs this permission (same with the task permissions)

Reply:
That's a good point - based on offline discussion I moved task scopes to its own canned policy. As part of https://github.com/raster-foundry/raster-foundry/issues/5262 I'm going to add a canned policy for groundwork users